etc/config/c.defaults.properties

```diff
@@ -11,6 +11,36 @@ stubRe=\bmain\b
 stubText=int main(void){return 0;/*stub provided by Compiler Explorer*/}
 supportsLibraryCodeFilter=true
 
+group.cerberus.compilers=cerberus:cerberus-cheri
+
+compiler.cerberus.name=Cerberus
```

Comment:
If we want to demonstrate configuration, then setting all these options at the `group` level will make them shared amongst all `cerberus`  compilers, making it very much less repetititve.